jit_preloader.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", ">= 5.2", "< 8"
+  spec.add_dependency "activerecord", ">= 7", "< 8"
```

Comment:
Quick note for posterity here, this change is because of this: https://github.com/clio/jit_preloader/pull/50/files#diff-131ca0e8e3eeec97a35249b47da9814f2d6d1a51fb7e99aaa6edb9e2d1e57bd8R36

In rails 6.x, preloader takes these arguments to `#preload`: https://github.com/rails/rails/blob/147557dc16a5bd8bab96f999f359a570232e1873/activerecord/lib/active_record/associations/preloader.rb#L87

In 7.x, takes them in `.initialize`: https://github.com/rails/rails/blob/b5630e232b95859c4171a285cd8f130e37d3c501/activerecord/lib/active_record/associations/preloader.rb#L96

So we change the dependency in this version of the gem to require activerecord >= 7. Older version was: https://github.com/clio/jit_preloader/blob/5c665394eaac0b283ce84dd4dcc065d353f0d9bc/jit_preloader.gemspec#L21
So it will get resolved by bundler for rails 5.2 to 6.x.